lib/python/Components/Timezones.py

```
@@ -290,6 +284,12 @@ def activateTimezone(self, zone, area, runCallbacks=True):
 		except (IOError, OSError) as err:
 			print "[Timezones] Error %d: Linking '%s' to '/etc/localtime'! (%s)" % (err.errno, file, err.strerror)
 			pass
+		try:
+			with open("/etc/timezone", "w") as fd:
+				fd.write("%s\n" % tz)
+		except (IOError, OSError) as err:
+			print "[Timezones] Error %d: Updating '/etc/timezone'! (%s)" % (err.errno, err.strerror)
+			pass
```

Pass is a no-op (no operation). You need to use when you don't have any other commend to use.
Since you are using print above pass, pass I think is useless.

---

You are absolutely correct.  That is a left over from when I thought I would ignore the error.  I will remove the "pass" when I next update the code and if there are no more changes needed I will remove it when I have some free time.